astrolabe/atlas_runner.py

```diff
@@ -338,7 +338,14 @@ def wait_for_idle(self):
         wanted_state = 'idle'
         last_notified = 0
         while timer.elapsed < timeout:
-            cluster_info = self.cluster_url.get().data
+            sleep(1.0 / self.config.polling_frequency)
```

Comment:
I was confused by this until I realized polling_frequency is actually a frequency and not a timespan like heartbeatFrequencyMS.

Reply:
Yeah, interval is typically more intuitive than frequency. Technically `heartbeatFrequencyMS` is a misnomer and should be called `heartbeatIntervalMS` because it defines an interval.